src/common/image.h

```diff
@@ -89,6 +89,18 @@ typedef struct dt_image_raw_parameters_t
   unsigned user_flip : 8; // +8 = 32 bits.
 } dt_image_raw_parameters_t;
 
+typedef enum dt_exif_image_orientation_t
+{
+  EXIF_ORIENTATION__NONE              = 1,
```

Comment:
For consistency, in the current code we do not use double underscore. So I'll do the same here and use a single underscore after ORIENTATION.

Reply:
You're right. Let me fix that